src/services/user/services/AdminUsers.js

```diff
@@ -310,16 +180,13 @@ const formatBirthdayOfUsers = ({ result: { data: users } }) => {
 const adminHookGenerator = (kind) => ({
 	before: {
 		all: [authenticate('jwt')],
-		find: [hasSchoolPermission(`${kind}_LIST`)],
-		get: [hasSchoolPermission(`${kind}_LIST`)],
 		create: [hasSchoolPermission(`${kind}_CREATE`), blockDisposableEmail('email')],
 		update: [hasSchoolPermission(`${kind}_EDIT`), protectImmutableAttributes, blockDisposableEmail('email')],
 		patch: [hasSchoolPermission(`${kind}_EDIT`), protectImmutableAttributes, blockDisposableEmail('email')],
 		remove: [hasSchoolPermission(`${kind}_DELETE`), parseRequestQuery, validateParams],
 	},
 	after: {
 		all: [transformToDataTransferObject],
-		find: [formatBirthdayOfUsers],
```

Comment:
If you remove this, you can also remove the definition of `formatBirthdayOfUsers `, because it is used nowhere else.